source/server/quickstart.rst

```diff
@@ -0,0 +1,147 @@
+=================
+Quick Start Guide
+=================
+
+This is a step by step reference on how to create a Minecraft server with Sponge.
+
+.. note::
+
+    Are you migrating from a Bukkit or Canary based server? Please read
+    :doc:`getting-started/migrating` as it explains the differences between these platforms and Sponge,
+    and explains the migration process.
```

Comment:
I think the second "explains" is redundant.

Reply:
I think you're right.

Reply:
Ah, hold on, I know why I put that. It's because of the comma before the last "and" to separate it from the "these platforms and Sponge" section. The statement would become (removing the comma as it makes less sense at this point)...

> as it explains the differences between these platforms and Sponge and the migration process

which would infer that I'm describing the differences between all three. Maybe changing "explains" to "sets out" or something similar will make it different enough to state that there are two points being made, rather than one amalgamated one?

> as it explains differences between these platforms and Sponge, and sets out the migration process

May also just need a complete reword.

Reply:
If you leave the comma in it seems less ambiguous, but it's not at all necessary to change it anyway. Better to merge it than make word-salad.